core-addon/DataCollection.js

```diff
@@ -188,7 +188,7 @@ module.exports = class DataCollection {
     const FIELD_MAPPING = {
       "age": "age",
       "gender": "gender",
-      "hispanicLatinoSpanishOrigin": "origin",
+      "hispanicLatinxSpanishOrigin": "origin",
```

Comment:
We sadly need to change [the schema for this]. :'( (We can't migrate soon enough..!)

Reply:
I just created [the PR for the schema](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/655).